google-home-community.groovy

```diff
@@ -1666,6 +1680,14 @@ private executeCommand_Reverse(deviceInfo, command) {
     return [:]
 }
 
+@SuppressWarnings(['UnusedPrivateMethod', 'UnusedPrivateMethodParameter'])
+private executeCommand_Locate(deviceInfo, command) {
+    checkMfa(deviceInfo.deviceType, "Locator", command)
```

Comment:
To go along with my other comment, I think `"Locate"` would be better here.

Reply:
Agreed.  Misunderstood use case for MFA, and this is removed.

Reply:
This still needs to call `checkMfa(deviceInfo.deviceType, "Locate", command)`.  In order to support MFA each command needs to call `checkMfa` with one of the strings in the trait's `commands` array.

Reply:
Misunderstood.  This has been corrected.  This particular trait is tagged as "Locator", requires references to such, but the command is "Locate".  Nice and confusing!